tracing/README.md

```diff
@@ -0,0 +1,188 @@
+// create a tracer from configuration
+Tracer tracer = TracerBuilder.create(config.get("tracing"))    
+    // build the tracer and register as global tracer
+    .buildAndRegister();
+```
+
+and associated configuration:
+```yaml
+tracing:
+  service: "myService"
+  host: "zipkin"
+```
+
+##Module `helidon-tracing-jersey-client`
+Integration with Jersey client to add tracing support
+ for outbound requests.
+ 
+The client is registered automatically with Jersey (e.g. no need to add any filter).
+
+There are two modes of usage:
+1. Use within a Jersey server (requires `helidon-tracing-jersey` with a registered filter, or `helidon-microprofile-tracing`)
+2. Standalone use
+
+###Use within Jersey server
+The server filters create a context for the client, so as long as the client is invoked
+within the same thread, no additional configuration is required - just use 
+client as usual and tracing will be propagated and added for outbound call.
+
+###Standalone use
+In case the Jersey client is used on its own, the tracing filter cannot obtain 
+information needed for outbound call and it has to be provided.
+
+Example of use:
+```java
+response = webTarget
+            .request()
+            // tracer information - not required if global tracer should be used
+            .property(ClientTracingFilter.TRACER_PROPERTY_NAME, tracer)
+            // the current tracing span context to be used as a parent for outbound request
+            // if not provided a new span with no parent would be created
+            .property(ClientTracingFilter.CURRENT_SPAN_CONTEXT_PROPERTY_NAME, spanContext)
+            .get();
+``` 
+
+##Module `helidon-tracing-jersey`
+Integration with "pure" Jersey server. This module should not be directly used when using
+Helidon MP, use `helidon-microprofile-tracing` instead.
+
+To configure tracing with Jersey, add `TracingFilter` to your application/resource configuration.
+The tracing filter will start a new span for each jersey call and register context for client calls.
+`helidon-tracing-jersey-client` is a transitive dependency of this module.
+
+##Module `helidon-microprofile-tracing`
+Provides automated integration with Helidon MP, including automated configuration of
+tracer and of server-side filters that register context for client calls and trace each 
+request (unless explicitly disabled using configuration)
+This module is located in /microprofile/tracing
+
+##Module `helidon-tracing-zipkin`
+Integration with Zipkin (https://zipkin.io/). Easiest approach is to use a docker image
+`zipkin` that, by default, runs on the expected hostname and port.
+
+When you add this module to classpath, the SPI is automatically picked up and Zipkin tracer 
+would be used.
+
+See class `ZipkinTracerBuilder` for documentation of supported configuration options. The classes
+in this module should not be used directly, use `helidon-tracing` module API instead (unless
+you want to create a hard source code dependency on Zipkin tracer).
```

Comment:
"The classes in this module should not be used directly, use `helidon-tracing` module API instead (unless you want to create a hard source code dependency on Zipkin tracer)."  --> I suggest rephrasing this as:

"Don't use the classes in this module directly, unless you want to create a hard source-code dependency on Zipkin tracer. Use the `helidon-tracing` module instead."